Packs/CommonScripts/Scripts/ContentPackInstaller/ContentPackInstaller.py

```diff
@@ -22,30 +22,33 @@ def __init__(self, instance_name: str = None):
 
         self.get_installed_packs()
 
-    def get_installed_packs(self) -> None:
-        """Gets the current installed packs on the machine.
-        """
-        demisto.debug(f'{SCRIPT_NAME} - Fetching installed packs from marketplace.')
-
-        args = {'uri': '/contentpacks/metadata/installed'}
-
+    def _call_execute_command(self, args, command):
```

Comment:
Let's be more aligned with the execute command function
```suggestion
    def _call_execute_command(self, command, args):
```